litgpt/api.py

```diff
@@ -252,7 +252,7 @@ def load(
     def distribute(
         self,
         accelerator: Literal["cpu", "cuda", "auto"] = "auto",
-        devices: Union[int, Literal["auto"]] = "auto",
+        devices: Union[int, List[int]] = 1,
```

Comment:
I think as per William's feedback, this should be "auto". Any particular reason why this was changed @apaz-cli ?

Reply:
@rasbt I think that a merge failed somehow. I'll clean these up.